pyhgvs/tests/genome.py

```diff
@@ -115,7 +119,7 @@ def read(self, filename):
 
         filename: a filename string or file stream.
         """
-        if isinstance(filename, basestring):
```

Comment:
@lucaswiman do you suggest a `six` usage here that allows python 2 unicode filenames?